jbmc/regression/jbmc/multidimensional-array-char/test.desc

```diff
@@ -0,0 +1,12 @@
+CORE
+My.class
+--function My.check2DCharArray --unwind 5 --max-nondet-array-length 10
+line 8 assertion at file My.java.*: FAILURE
+line 11 assertion at file My.java.*: FAILURE
```

Comment:
:pick: maybe add an assertion that should succeed, like maybe for instance `assert diff < 10`